### ISSUE TITLE
local_use_cli()

### ISSUE THREAD

**Issue by romainfrancois** [tidyverse/dplyr#6047]
```r
on_load(
  local_use_cli()
)
```

**Comment by romainfrancois** [comment]
@lionel- is it expected that when using `local_use_cli()` to enable `cli`, we'd lose follow up bullets in a "Caused by"  error ? 

``` r
library(rlang)

f <- function(use_cli = TRUE) {
  if (use_cli) local_use_cli()
  
  tryCatch(
    abort(c(
      "first line", 
      i = "second line", 
      x = "third line"
    )), 
    error = function(cnd) {
      abort("problem", parent = cnd)
    }
  )
}

tryCatch(f(TRUE), error = function(cnd) {
  print(cnd, backtrace = FALSE)
})
#> <error/rlang_error>
#> Error: 
#>   problem
#> Caused by error in `f()`: 
#>   first line
tryCatch(f(FALSE), error = function(cnd) {
  print(cnd, backtrace = FALSE)
})
#> <error/rlang_error>
#> Error: 
#>   problem
#> Caused by error in `f()`: 
#>   first line
#>   ℹ second line
#>   x third line
```

<sup>Created on 2021-11-09 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

**Comment by lionel-** [comment]
hmm I don't think so, I'll take a look.

**Comment by lionel-** [comment]
Weird I can reproduce with installed dev rlang but not with `load_all()`'d dev rlang.

**Comment by lionel-** [comment]
Oh of course, since rlang uses `local_use_cli()` it is always on.

A better reprex would be:

```r
f <- function(use_cli = TRUE) {
  local_use_cli(format = use_cli)

  tryCatch(
    abort(c(
      "first line",
      i = "second line",
      x = "third line"
    )),
    error = function(cnd) {
      abort("problem", parent = cnd)
    }
  )
}
```

**Comment by lionel-** [comment]
@romainfrancois Now fixed on dev rlang.

**Comment by romainfrancois** [comment]
Thanks. I'll follow up in #6075

**Comment by lionel-** [comment]
Closed by #6119
